airflow/utils/db_cleanup.py

```diff
@@ -123,6 +126,15 @@ def _check_for_rows(*, query: Query, print_rows=False):
     return num_entities
 
 
+def _dump_db(*, target_table, file_path, export_format, session):
```

Comment:
Should raise an exception if `export_format` is an unexpected value (future-proof).

Reply:
Added a test in argparse since the choices=('csv',) would prevent the incorrect format from making its way to this function